Scripts/Helpers/ProxyNode.cs

```diff
@@ -24,6 +25,9 @@ public class ProxyFeedbackRequest : FeedbackRequest
         public bool suppressExisting;
         public bool showBody;
         public float duration = 5f;
+        public int maxPresentations = 2;
```

Comment:
Should this be const or do we offer a supported way to change this value?

Reply:
This, and duration, can be set to anything you want when requesting feedback. I could put them in consts, but this is the only place that literal value is used, and I figured the variable name was enough.